src/styles/utils/variables.scss

```diff
@@ -0,0 +1,4 @@
+$main-text-color: #060b35;
+$sec-text-color: #616070;
+$btn-color: #00acdc;
```

Comment:
better use more descriptive name for colors for example $color-blue 
Because this color can be used in another place not only button)